neverbleed.c

```diff
@@ -596,6 +830,14 @@ static int load_key_stub(struct expbuf_t *buf)
     RSA *rsa = NULL;
     size_t key_index = SIZE_MAX;
     char *estr = NULL, *nstr = NULL, errbuf[NEVERBLEED_ERRBUF_SIZE] = "";
+    size_t type = NEVERBLEED_TYPE_NONE;
+    EC_KEY *ec_key = NULL;
+    EVP_PKEY *pkey;
```

Comment:
I see the following warnings when trying to compile the file with OpenSSL 1.0.2.

I assume that use of `pkey` without initialization in an error. Also, it might be preferable to suppress the unused warnings as disscussed on https://github.com/h2o/neverbleed/pull/11/files#r76543935.

```
neverbleed.c:837:21: warning: unused variable 'ec_pubkey' [-Wunused-variable]
    const EC_POINT *ec_pubkey;
                    ^
neverbleed.c:834:13: warning: unused variable 'ec_key' [-Wunused-variable]
    EC_KEY *ec_key = NULL;
            ^
neverbleed.c:847:9: warning: variable 'pkey' is used uninitialized whenever 'if' condition is true
      [-Wsometimes-uninitialized]
    if ((fp = fopen(fn, "rt")) == NULL) {
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
neverbleed.c:909:9: note: uninitialized use occurs here
    if (pkey != NULL)
        ^~~~
neverbleed.c:847:5: note: remove the 'if' if its condition is always false
    if ((fp = fopen(fn, "rt")) == NULL) {
    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
neverbleed.c:835:19: note: initialize the variable 'pkey' to silence this warning
    EVP_PKEY *pkey;
                  ^
                   = NULL
```